Libraries/oneTBB/tbb-task-sycl/src/tbb-task-sycl.cpp

```diff
@@ -0,0 +1,154 @@
+﻿//==============================================================
+// Copyright © 2019 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+#include <array>
+#include <iostream>
+
+#include <CL/sycl.hpp>
+
+#include <tbb/blocked_range.h>
+#include <tbb/global_control.h>
+#include <tbb/parallel_for.h>
+#include "tbb/task_group.h"
+
+using namespace cl::sycl;
+
+constexpr cl::sycl::access::mode sycl_read = cl::sycl::access::mode::read;
+constexpr cl::sycl::access::mode sycl_write = cl::sycl::access::mode::write;
+constexpr cl::sycl::access::mode sycl_read_write = cl::sycl::access::mode::read_write;
+
+
+// exception handler
```

Comment:
Same thing as the above comments. Please use the dp_common.hpp way of calling the exception handler

queue q(default_selector{}, dpc_common::exception_handler);